services/BasicTypes.proto

```diff
@@ -340,24 +340,50 @@ message CurrentAndNextFeeSchedule {
     FeeSchedule nextFeeSchedule = 2; // Contains next Fee Schedule
 }
 
+/* This contains the IP Address and the port representing an endpoint of a Node in a network */
+message NodeEndpoint {
+    string ipAddress = 1; // The IP address of the node
+    string port = 2; // The port of the node
+}
+
 /*
 The metadata for a Node – including IP Address, and the crypto account associated with the Node.
 */
 message NodeAddress {
```

Comment:
Need to add a comment that this structure is intended to be used to parse `0.0.102` and that its use to parse `0.0.101` is deprecated and to use the new message type.